### ISSUE TITLE
Not working in Coursera

### ISSUE THREAD

**Issue by lijovklm** [vantezzen/skip-silence#16]
Hello,

Thanks for making such a great add-on. I use it extensively on youtube.
Recently when I tried it on Coursera it doesn't work. Icon is enabled, but when I turn it on volume bar is empty. below screenshot. 
![image](https://user-images.githubusercontent.com/24433342/86099992-d4403700-bad5-11ea-8407-07729117d211.png)

However I did get it working a couple of times as well.
I am not sure why this works strange here.

**Comment by vantezzen** [comment]
Just tried this with Skip Silence 3 and Coursera and it looks like it works - can somebody confirm?

![Screen Recording 2020-11-07 at 11 12 55](https://user-images.githubusercontent.com/10333196/98438439-75757480-20ea-11eb-9ba1-5ef811943616.gif)

**Comment by lijovklm** [comment]
Yes. It works now. and what a coincidence, I am taking the same course as you are. 👍 
![image](https://user-images.githubusercontent.com/24433342/98637274-3e070200-234e-11eb-998d-82025eac8e7a.png)

**Comment by lijovklm** [comment]
However, I got stumbled upon a small issue, when moved to next video, it starts to skip the video in 10x which is my silent setting, or not work at all. In both cases I must disable and again enable to get it working.
